app/javascript/components/index.js

```diff
@@ -144,7 +144,7 @@ export const coreComponents = [
     data: true,
     store: false
   },
-  { name: 'v2v_ui_plugin', type: MiqV2vUi }
+  { name: 'manageiq-v2v', type: ManageIQV2V }
```

Comment:
This bit should just be the name of the react component, and the `name` only needs to match the `mount_react_component` invocation arg.

Isn't it confusing to call the root component the same as the repo? To me, something like `v2v-root` would sound a bit saner..

(But no strong feelings about it, just saying :).)

Reply:
🤷‍♂️  I don't understand any of this, tbh.  I'll pick whatever string you tell me to :smile:

Reply:
It's fine as it is, we can always rename later :)